pkg/registry/core/pod/storage/storage_test.go

```diff
@@ -676,12 +680,15 @@ func TestEtcdCreateWithContainersNotFound(t *testing.T) {
 		t.Fatalf("unexpected error: %v", err)
 	}
 
+	featuregatetesting.SetFeatureGateEmulationVersionDuringTest(t, utilfeature.DefaultFeatureGate, version.MustParse("1.33"))
```

Comment:
Is this call still required? I believe I had to add this previously due to the default version being n-1 (where n is the 'next' release), but I think that may have changed and we now make the default version be `n` (i.e. 'next') 👀